scripts/lc-builds/toss4_nvcc_clang.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright (c) 2016-25, Lawrence Livermore National Security, LLC
+# and RAJA project contributors. See the RAJA/LICENSE file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+
+if [[ $# -lt 3 ]]; then
+  echo
+  echo "You must pass 3 arguments to the script (in this order): "
+  echo "   1) compiler version number for nvcc"
+  echo "   2) CUDA compute architecture (number only, not 'sm_90' for example)"
+  echo "   3) compiler version number for clang"
+  echo
+  echo "For example: "
+  echo "    toss4_nvcc_clang.sh 12.9.1 90 19.1.3-magic"
+  exit
+fi
+
+COMP_NVCC_VER=$1
+COMP_ARCH=$2
+COMP_CLANG_VER=$3
+shift 3
+
+BUILD_SUFFIX=lc_toss4-nvcc${COMP_NVCC_VER}-${COMP_ARCH}-clang${COMP_CLANG_VER}
+
+echo
+echo "Creating build directory build_${BUILD_SUFFIX} and generating configuration in it"
+echo "Configuration extra arguments:"
+echo "   $@"
+echo
+
+rm -rf build_${BUILD_SUFFIX} >/dev/null
+mkdir build_${BUILD_SUFFIX} && cd build_${BUILD_SUFFIX}
+
+module load cmake/3.23.1
```

Comment:
Note the older version of CMake here. This is done because it is the latest version that is supported on both RZWHIPPET and RZADAMS. Newer versions of CMake on both machines do not match. Shall we keep this?